src/components/ItemDetail/DetailSummary/DownloadRow.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function handleDownload(file, width) {
+  const path = `${file}/full/${width},/0/default.jpg`;
+  window.location = path;
+}
+
+const DownloadRow = props => {
+  const file = props.item.representative_file_url;
+
+  return (
+    <div className="summary-list">
+      <div className="summary-list-item download">
+        <div className="data-label">Download Options</div>
+        <div className="data-value">
+          <button
+            className="download-option-box"
+            onClick={() => {
+              handleDownload(file, 300);
+            }}
```

Comment:
No biggie, but you may be able to trim this down to one line:
`onClick = {() => handleDownload(file, 300)}`